src/app/about/officer-card.tsx

```diff
@@ -0,0 +1,56 @@
+import Image from "next/image";
+import styles from "./OfficerCard.module.css";
+
+type SocialLink = {
+  name: string;
+  url: string;
+  icon: string;
+};
+
+export type OfficerCardProperties = {
```

Comment:
[Comment]: Could also name this `OfficerCardProps` since the official React name is `props` - not sure what the convention there is; but this is what we do normally and saves a few characters. No strong opinion, but we should consider this the convention for this project moving forward and be consistent.

Reply:
Can't change this due to eslint [unicorn/prevent-abbreviations](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/prevent-abbreviations.md). I have decided to modify `.eslintrc.js` to allow this.